imports/plugins/included/search-mongo/server/publications/searchresults.js

```diff
@@ -68,12 +71,29 @@ getResults.orders = function (searchTerm, facets, maxResults, userId) {
           $regex: searchTerm,
           $options: "i"
         } },
+        { billingCard: {
+          $regex: searchTerm,
+          $options: "i"
+        } },
+        // if parsed phone is empty, default to original, otherwise quering returns false matches
         { billingPhone: {
-          $regex: "^" + searchPhone + "$",
+          $regex: searchPhone || searchTerm,
```

Comment:
I'm fairly certain removing those regex markers is going to cause a problem

Reply:
I'll review this.